src/VueProgram.ts

```diff
@@ -334,8 +353,9 @@ export class VueProgram {
 
     // There is src attribute
     if (script.attrs.src) {
-      // import path cannot be end with '.ts[x]'
-      const src = script.attrs.src.replace(/\.tsx?$/i, '');
+      // Import path cannot end with '.ts[x]'
+      // In v3, `attrs` can be `true` (attribute without a value), but `src` should be a string
+      const src = (<string>script.attrs.src).replace(/\.tsx?$/i, '');
```

Comment:
What if someone will add the `src` attribute without value and then will save it? Wouldn't it throw an error inside the `fork-ts-checker-webpack-plugin`? (calling undefined `replace` method)

Reply:
Yes, it would. I was trying to fixup the api changes quickly and it's a bit sloppy.
I agree it would be better not to throw inside `fork-ts-checker-webpack-plugin`, even when user input makes no sense.
Instead of casting we could use the good old trick `src + ""` that turns anything into a string. If that string makes no sense (i.e. is not a real file)... see comment on lines 362-364.